tests/integration/security/authorization_test.go

```diff
@@ -1153,3 +1153,58 @@ func TestAuthorization_Audit(t *testing.T) {
 			rbacUtil.RunRBACTest(t, cases)
 		})
 }
+
+
+// TestAuthorization_External tests the authorization policy with action "EXTERNAL".
+func TestAuthorization_External(t *testing.T) {
+	framework.NewTest(t).
+		Features("security.authorization.external-action").
+		Run(func(ctx framework.TestContext) {
+			ns := namespace.NewOrFail(t, ctx, namespace.Config{
+				Prefix: "v1beta1-external",
+				Inject: true,
+			})
+
+			var a, b echo.Instance
+			echoboot.NewBuilder(ctx).
```

Comment:
we should refactor these tests to not create new echo deployments for each test, it makes adding a test like this take ~30s rather than ~1s. Can be done latter

Reply:
does this mean to run all tests in the same namespace? currently each test is running in its own namespace.